automated_packaging/prepare_changelog.pl

```diff
@@ -25,10 +25,10 @@ ()
 
 $github_token = get_and_verify_token();
 
-# Checkout the master branch
-`git checkout master`;
+# Checkout the main branch
+`git checkout main`;
```

Comment:
Reminder: this line causes issues if the local `main` branch is behind the tracked remote branch.